packages/components-css/hero/_mixin.scss

```
@@ -0,0 +1,77 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Community for NL Design System
+ */
+
+@mixin rhc-hero {
+  block-size: auto;
+  inline-size: var(--rhc-hero-inline-size, inherit);
+  overflow: hidden;
+  position: relative;
+}
+
+@mixin rhc-hero--aspect-ratio-16-9 {
+  aspect-ratio: 16 / 9;
+}
+
+@mixin rhc-hero--aspect-ratio-1-1 {
+  aspect-ratio: 1 / 1;
+}
+
+@mixin rhc-hero--aspect-ratio-4-3 {
+  aspect-ratio: 4 / 3;
+}
+
+@mixin rhc-hero__message {
+  --utrecht-heading-1-color: var(--rhc-hero-message-color, inherit);
+  --utrecht-heading-2-color: var(--rhc-hero-message-color, inherit);
+  --utrecht-heading-3-color: var(--rhc-hero-message-color, inherit);
+  --utrecht-heading-4-color: var(--rhc-hero-message-color, inherit);
+  --utrecht-heading-5-color: var(--rhc-hero-message-color, inherit);
+  --utrecht-heading-6-color: var(--rhc-hero-message-color, inherit);
+  --utrecht-paragraph-color: var(--rhc-hero-message-color, inherit);
```

Misschien kun je het ook simpeler doen:

```suggestion
  --utrecht-paragraph-color: currentColor;
```

---

Done